applications/blockops_parachain_node_deployment.md

```diff
@@ -0,0 +1,288 @@
+- Create New Project
+<a href="https://ibb.co/CmQwccg"><img src="https://i.ibb.co/Fmzhrr2/Parachain-flow03.jpg" alt="Parachain-flow03" border="0"></a>
+- Select Cloud or Dedicated Node Provider
+
+- Launch Configuration
+<a href="https://ibb.co/3vFVSDX"><img src="https://i.ibb.co/ScmHVhT/Parachain-flow05.jpg" alt="Parachain-flow05" border="0"></a>
+
+- Summary
+<a href="https://ibb.co/3hvZQqF"><img src="https://i.ibb.co/Th13dS0/Parachain-flow06.jpg" alt="Parachain-flow06" border="0"></a>
+
+  <a href="https://ibb.co/64QjQ5m"><img src="https://i.ibb.co/Vv6f68t/Parachain-flow07.jpg" alt="Parachain-flow07" border="0"></a>
+
+- Active Networks
+<a href="https://ibb.co/Y8NgnpH"><img src="https://i.ibb.co/P4rkXwq/Parachain-flow08.jpg" alt="Parachain-flow08" border="0"></a>
+
+- UI Mockups of Blockops Monitoring Implementation
+
+  <a href="https://ibb.co/9pKwqFs"><img src="https://i.ibb.co/NFD1L8Z/Monitoring-1.jpg" alt="Monitoring-1" border="0"></a>
+
+  <a href="https://ibb.co/CPW1n1b"><img src="https://i.ibb.co/GHFxVxW/Monitoring-2.jpg" alt="Monitoring-2" border="0"></a>
+
+  <a href="https://ibb.co/HKZfyN7"><img src="https://i.ibb.co/XWHrPyX/Monitoring-3.jpg" alt="Monitoring-3" border="0"></a>
+
+**An overview of the technology stack to be used**
+
+- Nodejs and Reactjs (backend and frontend framework)
+- TypeSript (frontend and backend scripts)
+- Golang
+- Docker
+- Kubernetes
+- Pulumi
+- Prometheus & Grafana
+
+
+### Ecosystem Fit
+
+- **Where and how does your project fit into the ecosystem?:** With the complexity involved in setting up, managing and maintaining parachain networks, this project with enable faster and easier parachain deployment making the technology more accessible and promoting rapid development and also provides dedicated optimized machines for substrate development.
+
+
+- **Who is your target audience?:** Our target audience are dApps developers looking to bootstrap their parachains network and get started quickly and substrate developers substrate developers looking to build and iterate faster as well as test their smart contracts codes against live networks with Beefy built for purpose dedicated machines.
+
+- **What need(s) does your project meet?:** This project will significantly enhance the efficiency and reliability of the development process within the Polkadot ecosystem.
+
+- **Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?:** OnFinality has been a successful and proven model for only single node deployment. However Onfinality doesn't offer bootstrappable node deployments for parachains. There is currently no out of the box advanced analytics and proactive monitoring to ensure a production grade incident management of networks and nodes. From that perspective, we are a unique solution that combines both services and offers additional interests to our users.
```

Comment:
[Gantree](https://github.com/gantree-io) is not actively maintained currently, but have you looked into what they have built already? Perhaps some of it could be reused. 
Also, are you familiar with what [Tanssi](https://www.tanssi.network/) is doing with regard to infrastructure?

Reply:
Thanks for sharing the Gantree reference, check them out and we can draw learnings from their implantation.

The common gooal with these two is that we are all trying to solve the same problems in different ways and so far we are closer with a Simpler working solution.